firmware/open_evse/EnergyMeter.cpp

```diff
@@ -10,11 +10,14 @@ EnergyMeter::EnergyMeter()
   m_bFlags = 0;
   m_wattSeconds = 0;
 
-  if (eeprom_read_dword((uint32_t*)EOFS_KWH_ACCUMULATED) == 0xffffffff) { // Check for unitialized eeprom condition so it can begin at 0kWh
-    eeprom_write_dword((uint32_t*)EOFS_KWH_ACCUMULATED,0); //  Set the four bytes to zero just once in the case of unitialized eeprom
+  // check for unitialized eeprom condition so it can begin at 0kWh
+  if (eeprom_read_dword((uint32_t*)EOFS_KWH_ACCUMULATED) == 0xffffffff) {
+    // set the four bytes to zero just once in the case of unitialized eeprom
+    eeprom_write_dword((uint32_t*)EOFS_KWH_ACCUMULATED,0);
   }
   
-  m_wattHoursTot = eeprom_read_dword((uint32_t*)EOFS_KWH_ACCUMULATED);        // get the stored value for the kWh from eeprom
+  // get the stored value for the kWh from eeprom
+  m_wattHoursTot = eeprom_read_dword((uint32_t*)EOFS_KWH_ACCUMULATED);
```

Comment:
No code changes here. I just moved the comments to their own line so you didn't have to scroll sideways to read them.